<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frameImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>681</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QScrollArea" name="scrollAreaIMAGE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>650</width>
       <height>490</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>648</width>
        <height>488</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonNEXT">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBACK">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOK">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>50</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonPositive">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>200</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Positive</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonNegative">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>230</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Negative</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTXTDir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>561</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditImageDirOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>561</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPositiveDir">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>550</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Positive Dir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNegativeDir">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>590</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Negative Dir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadVideo">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>570</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Video File</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSliderVIDEO">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>29</width>
      <height>521</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1187</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Frame Pos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPosFframeNumber">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>120</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSliderFrame">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>530</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>270</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PosX:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>300</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PosY:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>330</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Height Box:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>360</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Width Box: </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPosXi">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>270</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPosYi">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>300</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHeightBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWidthBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>360</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxINFOTXT">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>140</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStopVideo">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>570</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Registro desde el cual se empezara a numerar los frames en el archivo .txt que contiene la informacion de la posicion del frame positivo y negativo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Register Init</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSliderVIDEO</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelPosFframeNumber</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>860</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
